src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -45,105 +93,142 @@ export abstract class PageHelper {
     return $('.datatable-footer-inner.selected-count');
   }
 
-  getTitleText() {
-    let title;
-    return browser
-      .wait(() => {
-        title = $('.panel-title');
-        return title.isPresent();
-      })
-      .then(() => title.getText());
-  }
-
-  getTableCell(content) {
+  // getTitleText() {
```

Comment:
remove this block?

Reply:
I commented it out when I started and it seems it wasn't used anywhere, so yes, I'll remove it.